bundle/language_es.properties

```diff
@@ -492,6 +492,7 @@ angal.lab.pleaseselectapatient
 angal.lab.positive                                                               = Positivo
 angal.lab.printreport                                                            = Imprimir reporte
 angal.lab.printtable                                                             = Imprimir tabla
+angal.lab.print																	 = Imprimir
```

Comment:
convertion space->tabs happening here

Reply:
Fixed with the push c9b6c019656dba1594baf288c36cf9d381f4130d